src/main/java/com/containers/controller/ContainerController.java

```diff
@@ -16,64 +16,53 @@ public ContainerController(ContainerService containerService) {
     }
 
-    @PostMapping(value = "/containers/newCon")
-    public String saveContainer(@ModelAttribute Container container) {
-        Container savedContainer = containerService.saveContainer(container);
-        return "redirect:/containers/find" + savedContainer.getContainerIdNumber();
+    @GetMapping("/update/{noContainer}")
+    public ModelAndView updateContainerView(@PathVariable String noContainer) {
+        ModelAndView modelAndView = new ModelAndView("addContainer");
+        modelAndView.addObject(containerService.findContainerById(noContainer));
+        modelAndView.addObject("update", true);
+        return modelAndView;
     }
 
-    @GetMapping("/containers/add/model")
-    public ModelAndView containersAddModel() {
-        ModelAndView modelAndView = new ModelAndView("containerModel");
-        return new ModelAndView("containerModel");
+    @PostMapping("/update")
+    public String updateContainer(@ModelAttribute Container container) {
+        containerService.updateContainer(container);
+        return "redirect:/containers/";
     }
 
-    @GetMapping(value = "/containers/find")
-    public ModelAndView findContainer(@RequestParam String containerNo) {
-        ModelAndView modelAndView = new ModelAndView("findContainer");
-        if (containerNo == null) {
-            return modelAndView.addObject("findContainer", containerService.findAllContainers());
-        } else
-            return modelAndView.addObject("findContainer", containerService.findContainerById(containerNo));
+    @PostMapping("/add")
+    public String addContainer(@ModelAttribute Container container) {
+        containerService.saveContainer(container);
+        return "redirect:/containers/";
     }
 
-//    @GetMapping("/containers/find/{id}")
-//    public ModelAndView findContainerById(@PathVariable String id) {
-//        ModelAndView modelAndView = new ModelAndView("findContainer");
-//        Container container = containerService.findContainerByIdAfterAdd(id);
-//        modelAndView.addObject(container);
-//        return modelAndView;
-//    }
-
-    @PostMapping("/containers/update")
-    public ModelAndView updateContainer(@ModelAttribute Container container) {
-        ModelAndView modelAndView = new ModelAndView("updateContainer");
-        modelAndView.addObject("updateContainer", containerService.updateContainer(container));
-        return modelAndView;
-    }
 
-    @GetMapping("/containers/delete")
-    public String deleteContainer(@RequestParam String containerNo) {
+    @GetMapping("/delete/{containerNo}")
+    public String deleteContainer(@PathVariable String containerNo) {
         containerService.deleteContainer(containerNo);
-        return "redirect:/containers";
-    }
-
-    @GetMapping("/container/status")
-    public ModelAndView getDamageStatus() {
-        return new ModelAndView("containerStatus");
+        return "redirect:/containers/";
     }
+//
+//    @GetMapping("/containers/add/model")
```

Comment:
Dlaczego to jest zakomentowane?

Reply:
chciałbym to przenieść do RaportController